cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,16 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://conduit.mate.academy/user/login');
+    cy.get('h1')
+      .should('contain.text', 'Sign in');
+    cy.get(':nth-child(1) > .form-control')
+      .type('luckezcy@gmail.com');
+    cy.get(':nth-child(2) > .form-control')
```

Comment:
Avoid using selectors like ':nth-child'. These are fragile and can break easily if the structure of your HTML changes. It's better to use more descriptive selectors such as 'data-test' attributes or class names. This makes your tests more robust and maintainable.